jpa/subsystem/src/main/java/org/jboss/as/jpa/container/AbstractEntityManager.java

```diff
@@ -74,7 +74,7 @@
     protected abstract boolean isExtendedPersistenceContext();
 
     /**
-     * @return true if a JTA transaction is active
+     * @return true if a Jakarta Transactions active
```

Comment:
The search and replace of `JTA transaction` with `Jakarta Transactions` has resulted in some awkward grammar (in this example it has produced a plural where a singular was required, searching for `Jakarta Transactions` should highlight cases where this has happened - there are quite a few of instances, particularly in the java source files).

Also there are many examples in the document where `JTA transaction is` has been replaced with `Jakarta Transactions`. These should all read `Jakarta Transaction is`.

Reply:
Thank you @mmusgrov for clearing the doubt of mine. In some of the places, I have modified the grammar so that it'll not look bad. But, then I was sceptical to use `Jakarta Transaction` instead of 'Jakarta Transactions` because I thought maybe It'll go against the specs naming convention. 
Thanks, I'll change it accordingly now :)

Reply:
Perhaps something like a "Jakarta Transactions transaction" - more then like a replace of just "JTA" with "Jakarta Transactions"

Reply:
I think that "Jakarta Transactions transaction" would be the proper option here, despite it being verbose.

Reply:
okay!

Reply:
Some alternatives:
1. `@return true if a (Jakarta Transactions) transaction is active`
2. `@return true if a Jakarta Transactions transaction is active`

IMO, we should be consistent with the style approach we use elsewhere, so IMO, @RanabirChakraborty  my vote is for you to pick which form based on what you are doing elsewhere.  

Does anyone else feel strongly about #1 versus #2?  I'm not saying that you cannot vote also, I'm just deferring my vote. :)

Reply:
I am unclear what the parentheses do in Javadoc in this area, so I am hesitant to vote.

Reply:
IMO, from the readers perspective, they would read `return true if transaction is active` where the parentheses is clarifying that the transaction happens to be a `Jakarta Transactions transaction`.

Reply:
Got you now, thanks! I wondered if it was something understood by Javadoc.

Given that, I don't feel strongly enough to vote and I think either would work for me. I guess the established convention in the doc before was to not distinguish the source of the transaction in parenthesis (e.g. "(JTA) transaction").

Reply:
Following Tom and Alessio's call already changed to `Jakarta Transactions transaction` which I guess goes with your option 2.